src/labelformat/formats/lightly.py

```diff
@@ -62,6 +69,12 @@ def get_labels(self) -> Iterable[ImageObjectDetection]:
             if json_path.name == "schema.json":
                 continue
             data = json.loads(json_path.read_text())
+            if data["file_name"] not in filename_to_image:
+                if self._skip_labels_without_image:
+                    continue
```

Comment:
Optional:
```suggestion
            if (
                data["file_name"] not in filename_to_image
                and not self._skip_labels_without_image
            ):
                raise ...
```

Reply:
Actually the suggestion is wrong, the former code is correct.

Reply:
We need to keep `continue` to skip to the next file. Otherwise we'll end up with the `KeyError` again.